packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+        </div>
+      </div>
+    </div>
+  </div>
+</figure>
+
+```html
+    <div class="ods-tabs" id="example-0">
+      <div class="ods-tablist" role="tablist" aria-label="">
+        <button class="ods-tab" id="tablistitem-0-0" role="tab" aria-selected="false" aria-controls="tab-0-0">
+          Agents
+        </button>
+        <button class="ods-tab" id="tablistitem-0-1" role="tab" aria-selected="false" aria-controls="tab-0-1">
+          People
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-2">
+          Settings
+        </button>
+        <button class="ods-tab" id="tablistitem-0-2" role="tab" aria-selected="false" aria-controls="tab-0-3">
+          Import
+        </button>
+      </div>
+      <div class="ods-tabpanel">
+        <div id="tab-0-0" tabindex="0" role="tabpanel" aria-labelledby="tablistit-0-0">
+          <p>Tab Panel 0: Agents</p>
+        </div>
+        <div id="tab-0-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-1" hidden="">
+          <p>Tab Panel 1: People</p>
+        </div>
+        <div id="tab-0-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-2" hidden="">
+          <p>Tab Panel 2: Settings</p>
+        </div>
+        <div id="tab-0-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-0-3" hidden="">
+          <p>Tab Panel 3: Import</p>
+        </div>
+      </div>
+    </div>
+  </div>
+```
+## Anatomy
+
+### Tablist
+The tablist is the parent element that houses tabs. It provides no style in and of itself.
+
+### Tab
+The tab is the element in which the user clicks to change to the corresponding tabpanel.
+
+### Tab indicator
+The tab indicator is a pseudo-element used to indicate the active tab. Its position changes by calculating the left offset, as well as the width of the active tab element. (See [Switching Tabs](#switching-tabs))
+
+### Tabpanel
+A tabpanel is a simple container element. For each tab, there is an associated tab panel which contains the content associated with that tab.
+
+## Switching tabs
+
+When a tab is selected, three things must happen:
+
+1. The indicator must change position and width.
+2. The selected tab button's `aria-selected` attribute is `true`. If a different tab was previously selected, that tab button's `aria-selected` attribute must is `false`.
+3. The tabpanel corresponding to the tab button is shown. This is done by removing the `hidden` attribute on the tabpanel. If a different tabpanel was previously visible, the `hidden` attribute is applied to it.
+
+### CSS custom properties
+The tab indicator's left position and width are changed by updating CSS custom properties on the `ods-tabs` element. In the following example, you can see the tab indicator position is on the second element:
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <div class="ods-tabs" id="example-1" style="
+      --ods-indicator-width:97px;
```

Comment:
I don't mind this in production, but the parser won't catch it correctly if we multi-line attribute content.

Reply:
Which parser? Seems to render fine in hexo and have the correct syntax highlighting in prism